package.json

```diff
@@ -19,6 +19,7 @@
     "bl": "^2.0.1",
     "clone-deep": "^4.0.0",
     "color": "^3.0.0",
+    "gateway-addon": "https://github.com/mozilla-iot/gateway-addon-node",
```

Comment:
This is unnecessary and will potentially break things moving forward. Let's stick with the one bundled with the gateway.

Also, further down in this file, you should change the min API level to 2.